Issue #281 does not disable cram tests properly

```
[INFO]  Running Cram command line tests
[ERROR] Cram tests failed!
[ERROR] Usage: cram [OPTIONS] TESTS...
[ERROR] See: '/Users/arcivanov/Documents/src/my/pyscript/target/reports/cram.err' for details
Traceback (most recent call last):
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/cli.py", line 403, in main
    environments=options.environments, tasks=arguments)
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/reactor.py", line 129, in build
    self.build_execution_plan(tasks, execution_plan)
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/reactor.py", line 171, in build_execution_plan
    reactor=self)
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 336, in execute_execution_plan
    summaries.append(self.execute_task(task, **keyword_arguments))
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 283, in execute_task
    task.execute(self.logger, keyword_arguments)
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 168, in execute
    executable.execute(argument_dict)
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 99, in execute
    self.callable(*arguments)
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/plugins/python/cram_plugin.py", line 124, in run_integration_tests
    run_cram_tests(project, logger)
  File "/Users/arcivanov/.pyenv/versions/3.5.0/envs/my-venv/lib/python3.5/site-packages/pybuilder/plugins/python/cram_plugin.py", line 116, in run_cram_tests
    raise BuildFailedException("Cram tests failed!")
pybuilder.errors.BuildFailedException: Cram tests failed!
------------------------------------------------------------
BUILD FAILED - Cram tests failed!
------------------------------------------------------------
```

--- a/src/main/python/pybuilder/plugins/python/cram_plugin.py
+++ b/src/main/python/pybuilder/plugins/python/cram_plugin.py
@@ -85,6 +85,8 @@ def run_cram_tests(project, logger):
     if not cram_tests or len(cram_tests) == 0:
         if project.get_property("cram_fail_if_no_tests"):
             raise BuildFailedException("No Cram tests found!")
+        else:
+            return
 
     command_and_arguments = _cram_command_for(project)
     command_and_arguments.extend(cram_tests)

--- a/src/unittest/python/plugins/python/cram_plugin_tests.py
+++ b/src/unittest/python/plugins/python/cram_plugin_tests.py
@@ -31,7 +31,6 @@ from pybuilder.plugins.python.cram_plugin import (
 
 
 class CramPluginTests(unittest.TestCase):
-
     def test_command_respects_no_verbose(self):
         project = Project('.')
         project.set_property('verbose', False)
@@ -187,3 +186,73 @@ class CramPluginTests(unittest.TestCase):
                                 ]
         self.assertEquals(expected_info_calls, logger.info.call_args_list)
         self.assertEquals(expected_error_calls, logger.error.call_args_list)
+
+    @patch('pybuilder.plugins.python.cram_plugin._cram_command_for')
+    @patch('pybuilder.plugins.python.cram_plugin._find_files')
+    @patch('pybuilder.plugins.python.cram_plugin._report_file')
+    @patch('os.environ')
+    @patch('pybuilder.plugins.python.cram_plugin.read_file')
+    @patch('pybuilder.plugins.python.cram_plugin.execute_command')
+    def test_running_plugin_no_failure_no_tests(self,
+                                                execute_mock,
+                                                read_file_mock,
+                                                os_mock,
+                                                report_mock,
+                                                find_files_mock,
+                                                command_mock
+                                                ):
+        project = Project('.')
+        project.set_property('verbose', True)
+        project.set_property('dir_source_main_python', 'python')
+        project.set_property('dir_source_main_scripts', 'scripts')
+        project.set_property("cram_fail_if_no_tests", False)
+        logger = Mock()
+
+        command_mock.return_value = ['cram']
+        find_files_mock.return_value = []
+        report_mock.return_value = 'report_file'
+        os_mock.copy.return_value = {}
+        read_file_mock.return_value = ['test failes for file', '# results']
+        execute_mock.return_value = 1
+
+        run_cram_tests(project, logger)
+
+        execute_mock.assert_not_called()
+        expected_info_calls = [call('Running Cram command line tests'),
+                               ]
+        self.assertEquals(expected_info_calls, logger.info.call_args_list)
+
+    @patch('pybuilder.plugins.python.cram_plugin._cram_command_for')
+    @patch('pybuilder.plugins.python.cram_plugin._find_files')
+    @patch('pybuilder.plugins.python.cram_plugin._report_file')
+    @patch('os.environ')
+    @patch('pybuilder.plugins.python.cram_plugin.read_file')
+    @patch('pybuilder.plugins.python.cram_plugin.execute_command')
+    def test_running_plugin_failure_no_tests(self,
+                                             execute_mock,
+                                             read_file_mock,
+                                             os_mock,
+                                             report_mock,
+                                             find_files_mock,
+                                             command_mock
+                                             ):
+        project = Project('.')
+        project.set_property('verbose', True)
+        project.set_property('dir_source_main_python', 'python')
+        project.set_property('dir_source_main_scripts', 'scripts')
+        project.set_property("cram_fail_if_no_tests", True)
+        logger = Mock()
+
+        command_mock.return_value = ['cram']
+        find_files_mock.return_value = []
+        report_mock.return_value = 'report_file'
+        os_mock.copy.return_value = {}
+        read_file_mock.return_value = ['test failes for file', '# results']
+        execute_mock.return_value = 1
+
+        self.assertRaises(BuildFailedException, run_cram_tests, project, logger)
+
+        execute_mock.assert_not_called()
+        expected_info_calls = [call('Running Cram command line tests'),
+                               ]
+        self.assertEquals(expected_info_calls, logger.info.call_args_list)

